src/common/event.cpp

```diff
@@ -1226,7 +1226,7 @@ wxEvtHandler::~wxEvtHandler()
             delete entry->m_callbackUserData;
             delete entry;
         }
-        delete m_dynamicEvents;
+        m_dynamicEvents.reset();
```

Comment:
Do we even need this?

Reply:
You're right, that's a dtor, so `.reset()` is redundant.

Fixed in https://github.com/wxWidgets/wxWidgets/pull/25336/commits/981aba5174615695c5aae259d9c3236b53f2162d